### PR TITLE
Bump to v4.6.0-rc1

### PR DESCRIPTION
**Pull request by ashley-cui**
Woke up in a cold sweat realizing I accidentally bumped to v4.6.0 instead of v4.6.0-rc1, glad I caught it before the I pushed the tag.... This is why I shouldn't rush a release in on a Friday.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by ashley-cui** [comment]
@containers/podman-maintainers PTAL, probably more on the urgent side.

**Review by vrothberg — APPROVED** [review]
LGTM

Back to bed :)

**Review by Luap99 — APPROVED** [review]
/lgtm
/hold

**Comment by lsm5** [comment]
I guess @ashley-cui 's  on PTO. I can push the actual tag and create a pre-release

**Comment by mheon** [comment]
@lsm5 Sure, please do

**Comment by lsm5** [comment]
https://github.com/containers/podman/releases/tag/v4.6.0-rc1

**Pull request merged**
